app/mailers/school_mailer.rb

```diff
@@ -109,6 +111,7 @@ def partnered_school_invite_sit_email(
       rails_mailer: mailer_name,
       rails_mail_template: action_name,
       personalisation: {
+        school_name: school_name,
```

Comment:
I hope we don't have any delayed jobs trying to send this email, cause notify will break for them (one bad thing), and cause this change will break them too.

Reply:
This should be fine. It's run manually and was sometime last week.